app/views/my_modules/protocols/_protocol_status_bar.html.erb

```diff
@@ -18,7 +18,7 @@
     <div class="dropdown-content">
         <div class="dropdown-header">
           <h2 class="protocol-name">
-            <%= @protocol.parent&.name || @protocol.name %>
+            <%= t("labels.archived") if @protocol.archived? || @protocol.parent&.archived? %>&nbsp;<%= @protocol.parent&.name || @protocol.name %>
```

Comment:
I think it would be better to implement a "name_with_label" method with similar logic, and put it in the archivable concern. This way any archivable model will have this method available for use, and we can avoid a lot of duplication.

Also, use the archived_branch? method, to avoid manually checking the parent.